hdl/projects/grapefruit/fmc_ila.tcl

```diff
@@ -1,108 +1,399 @@
 create_debug_core u_ila_0 ila
```

Comment:
I think this file was erroneously checked in?

Reply:
this file is the ila, maybe poorly named but is used in the BUCK build and does need to be checked in